tests/supervisor/test_execution.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+
+import pytest
+from aleph_message.models import ItemHash
+
+from aleph.vm.conf import settings
+from aleph.vm.models import VmExecution
+from aleph.vm.storage import get_message
+
+
+@pytest.mark.asyncio
+async def test_create_execution():
+    """
+    Create a new VM execution and check that it starts properly.
+    """
+
+    settings.FAKE_DATA_PROGRAM = settings.BENCHMARK_FAKE_DATA_PROGRAM
+    settings.ALLOW_VM_NETWORKING = False
+    settings.USE_JAILER = False
+
+    # Ensure that the settings are correct and required files present.
+    settings.setup()
+    settings.check()
+
+    vm_hash = ItemHash("fake-hash-fake-hash-fake-hash-fake-hash-fake-hash-fake-hash-hash")
+    message = await get_message(ref=vm_hash)
+
+    execution = VmExecution(
+        vm_hash=vm_hash,
+        message=message.content,
+        original=message.content,
+        snapshot_manager=None,
+        systemd_manager=None,
+        persistent=False,
+    )
+
+    # Downloading the resources required may take some time, limit it to 10 seconds
+    await asyncio.wait_for(execution.prepare(), timeout=30)
+
+    execution.create(vm_id=3, tap_interface=None)
+    # await asyncio.wait_for(execution.start(), timeout=30)
```

Comment:
I think that test the start method will be needed here also to ensure that covers the main use case.